Automattic/woocommerce-payments
# Add developer document for "Version Support Policy"

Fixes #1775

#### Changes proposed in this Pull Request

- Add developer document for "Version Support Policy". 
- Also add the introduction in the main README.md.









#### Testing instructions





* Review the content of the new doc and ensure they're correct and helpful for contributors (developers).
* Recommend using this rendered link to review https://github.com/Automattic/woocommerce-payments/blob/add/version-support-policy/docs/version-support-policy.md

-------------------


**Post merge**
